src/types/payload.ts

```diff
@@ -0,0 +1,6 @@
+export interface Payload {
```

Comment:
Gdzie to jest wykorzystywane bo nie mogę znaleźć? Ta nazwa interfejsu nic nie mówi